rep/db/db.py

```diff
@@ -35,6 +35,16 @@ def create_db(self):
                 FOREIGN KEY (sourceType) REFERENCES raw_vote_object_source_type(type), 
                 FOREIGN KEY (format) REFERENCES raw_vote_object_format(format)
             );
+            
+        CREATE TABLE IF NOT EXISTS processed_vote_results
+            (
+                year INTEGER NOT NULL,
+                date VARCHAR NOT NULL,
```

Comment:
what format is the date actually being stored in? I'd rather this be an `INTEGER` and have the date stored in unix time. Otherwise it's going to be annoying to have to convert back and forth to an ISO 8601 date format or some other format that we agree on. Date math is also really easy when you're dealing with dates in unix time, so fetching records that are before/after a certain date in SQL is just a greater than/less than comparison

Reply:
I can look at converting it to unix time, right now it's in a really awkward format, like "3_12" for March 12th, then the year is in a separate column. I can look at combining the date and year into one column and representing it in unix time